database/seeders/ApiEndpointsSeeder.php

```diff
@@ -5822,6 +5822,21 @@ private function seedSummitEndpoints()
                     IGroup::TrackChairsAdmins,
                 ]
             ],
+            // track chair rating
```

Comment:
i do think that is missing the delete
( un cast rating)

Reply:
resolved at service level as discussed